src/ResourceManager/Network/Commands.Network/help/Add-AzureRmExpressRouteCircuitPeeringConfig.md

```diff
@@ -78,6 +80,19 @@ Accept pipeline input: True (ByValue)
 Accept wildcard characters: False
 ```
 
+### -LegacyMode
+The legacy mode of the Peering```yaml
```

Comment:
@EvgenyAgafonchikov please move the `` ```yaml `` text block at the end of this line to the next line and make sure that it is by itself. This will allow the markdown to be processed correctly.